src/css/body-content.scss

```diff
@@ -0,0 +1,65 @@
+@import "colors";
+.box{
+    margin-bottom: 30px;
+    margin-top: 22px;
+    float: left;
+    display: flex;
+    flex-direction: column;
+
+    &__title{
+        font-size: 22px;
+        line-height: 1.3em;
+        color: $red;
+        text-align: left;
+        margin-bottom: 6px;
+        padding-bottom: 5px;
+        font-weight: 600;
+    } 
+    &__link{
+        &__photo{ 
```

Comment:
are you sure about this: block - element - element?

http://getbem.com/naming/

here I think __link selector is unneeded